schematic/models/validate_attribute.py

```diff
@@ -1022,13 +1025,15 @@ def regex_validation(
                 f" They should be provided as follows ['regex', 'module name', 'regular expression']"
             )
 
-        errors = []
-        warnings = []
+        errors: list[list[str]] = []
+        warnings: list[list[str]] = []
 
         validation_rules = self.dmge.get_node_validation_rules(
             node_display_name=manifest_col.name
         )
-
+        # It seems like this statement can ever be true
```

Comment:
```suggestion
        # It seems like this statement can never be true
```

Reply:
Does this need a follow up tech debt ticket? Probably not something we would care to go back and fix if we'll sunset all this code anyways.